SolveMuscleRedundancy_FtildeState_CasADi.m

```diff
@@ -119,6 +119,18 @@
     end
 end
 
+% adapt the time indexes based on time vector IK solution
```

Comment:
I think it is indices rather than indexes, not sure though. This holds for all similar m files.